spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/RDDFunctions.scala

```diff
@@ -159,14 +158,39 @@ class RDDFunctions[T](rdd: RDD[T]) extends WritableToCassandra[T] with Serializa
   implicit
     connector: CassandraConnector = CassandraConnector(sparkContext.getConf),
     currentType: ClassTag[T],
-    rwf: RowWriterFactory[T]) = {
+    rwf: RowWriterFactory[T]): CassandraPartitionedRDD[T] = {
+
+    val converter = ReplicaLocator[T](connector, keyspaceName, tableName, partitionKeyMapper)
```

Comment:
You renamed ReplicaMapper to ReplicaLocator, but I still wonder, why this variable (and other parameters) are named `converter`. Why not just `replicaLocator`?